doc/tasks/doctrine_orm.md

```diff
@@ -0,0 +1,32 @@
+# Doctrine ORM
+
+The Doctrine ORM task will validate that your Doctrine mapping files and check if the mapping is in sync with the database.
+It lives under the `doctrine` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        doctrine_orm:
+            skip_mapping: false
+            skip_sync: true
+            triggered_by: ['php', 'xml', 'yml']
+```
+
+**skip_mapping**
+
+*Default: false*
+
+With this parameter you can skip the mapping validation check.
+
+**skip_sync**
```

Comment:
Changed it, having the defaults of the CLI tool makes sense. As I wrote in the PR it's just a personal preference of mine, don't really care if the DB is in sync when I commit and on CI the DB is always migrated anyway :)